### ISSUE TITLE
Setting parentStyle on a style causes exception

### ISSUE THREAD

**Issue by reubenfirmin** [rjaros/kvision#258]
```
    val baseCardStyle = Style {
// ...
    }

    val decoratedCardStyle = Style(parentStyle = baseCardStyle) {
// ...
    }
 ```

Applying decoratedCardStyle to an element leads to:

```
class.js?28dc:19 Uncaught DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('kv_styleclass_72 .kv_styleclass_73') contains HTML space characters, which are not valid in tokens.
    at Array.updateClass (webpack-internal:///../../node_modules/snabbdom/modules/class.js:19:50)
    at createElm (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:93:30)
    at addVnodes (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:122:45)
    at updateChildren (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:239:17)
    at patchVnode (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:266:21)
    at updateChildren (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:192:17)
    at patchVnode (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:266:21)
    at updateChildren (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:192:17)
    at patchVnode (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:266:21)
    at updateChildren (webpack-internal:///../../node_modules/snabbdom/es/snabbdom.js:192:17)
```

**Comment by reubenfirmin** [comment]
Seems that the issue is specifically the jump from kotlin 1.4.31 to kotlin 1.4.32; downgrading just kotlin resolves the issue.

That said is snabdom provided by kvision or by kjs?

**Comment by reubenfirmin** [comment]
Appears from https://developer.mozilla.org/en-US/docs/Web/API/Element/classList that the classList property should be used if there are multiple classes.

Alternately, maybe just merge the styles in kotlin and output a single class?

**Comment by rjaros** [comment]
It's definitely a bug in KVision. It has always been there. I just can't believe it shows up on a specific Kotlin version :-)

**Comment by reubenfirmin** [comment]
Yeah I figured that out. Slow build cycle here when dependencies change because I have bad internet.

I made:

```    
    fun Style.inheritFrom(other: Style): Style {
        val otherStyles = other.getSnStyle()
        val myStyles = getSnStyle()
        otherStyles.forEach {
            setStyle(it.first, it.second)
        }
        myStyles.forEach {
            setStyle(it.first, it.second)
        }
        return this
    }
```

...which seems to give me 90% of what I need (of course it doesn't respect pseudoclasses etc).

**Comment by rjaros** [comment]
Fixed in 4.5.0
